docs/reference/announcements-release-notes/880/whats-new-in-88.md

```diff
@@ -118,6 +118,13 @@ The Orchestration Cluster [Identity](/components/identity/identity-introduction.
 With this 8.8 change, the source of truth for Identity and Access Management for the Orchestration Cluster (including Zeebe, Operate, Tasklist, and its APIs) is now the Orchestration Cluster itself. This removes the reliance on the separate [Management Identity](/self-managed/components/management-identity/overview.md) (formerly "Identity") component.
 :::
 
+### Tenant interceptors
```

Comment:
Probably worth adding this and breaking changes to the announcements page as well 👍

Reply:
Added :)